Find a functionally equivalent code:com.android.internal.widget.LockPatternUtils.computePasswordQuality:COMMENT
Method Modifier: public      static      internal    
Comment:/**
 * Compute the password quality from the given password string.
 */

@@ -16,22 +16,22 @@
         return DevicePolicyManager.PASSWORD_QUALITY_ALPHABETIC;
     }
     if (hasDigit) {
-        return DevicePolicyManager.PASSWORD_QUALITY_NUMERIC;
+        return maxLengthSequence(password) > MAX_ALLOWED_SEQUENCE ? DevicePolicyManager.PASSWORD_QUALITY_NUMERIC : DevicePolicyManager.PASSWORD_QUALITY_NUMERIC_COMPLEX;
     }
     return DevicePolicyManager.PASSWORD_QUALITY_UNSPECIFIED;
 }

